libr/core/project.c

```diff
@@ -521,7 +521,7 @@ static bool store_files_and_maps (RCore *core, RIODesc *desc, ut32 id) {
 	RListIter *iter;
 	RIOMap *map;
 	if (desc) {
-		r_cons_printf ("ofs %s %s\n", desc->uri, r_str_rwx_i (desc->perm));
+		r_cons_printf ("\"ofs \\\"%s\\\" %s\"\n", desc->uri, r_str_rwx_i (desc->perm));
```

Comment:
> The whole command should be quoted instead but that wont work because you need to pass a second argument

Like this?

Reply:
no.. you should use the api, i dont think there's a way for ofs to work with paths with spaces

Reply:
It uses r_str_argv() so it works fine with just the inner quotes:

![pic](https://user-images.githubusercontent.com/23562137/53936213-e3e97000-4087-11e9-859e-6f95af175531.png)

I think the outer ones are unnecessary imho, should I revert it?

Reply:
ok

Reply:
then its fine

Reply:
as long as it works i will merge. we need to rework the command parser at some point to handler str_argv everywhere and such